### ISSUE TITLE
Duplicating Google Maps API with "beforeInteractive" script component

### ISSUE THREAD

**Issue by emetelyov** [vercel/next.js#28036]
### What version of Next.js are you using?

11.1.0

### What version of Node.js are you using?

15.7.0

### What browser are you using?

Chrome

### What operating system are you using?

macOS

### How are you deploying your application?

next start

### Describe the Bug

When I trying to add google initial script through the Script component it's loads twice and the console log shows an error:

`You have included the Google Maps JavaScript API multiple times on this page. This may cause unexpected errors.`

**_app.js**

`function MyApp({ Component, pageProps }) {
  return (
    <ChakraProvider theme={theme}>
      <Script
        strategy="beforeInteractive"
        src="https://maps.googleapis.com/maps/api/js?key=***&libraries=places"
      />
      <Component {...pageProps} />
    </ChakraProvider>
  );
}`

### Expected Behavior

For normal work, I need that script loads only once in the header before initializing.

### To Reproduce

Just add the line to _app.js:

`<Script
        strategy="beforeInteractive"
        src=""
      />`

**Comment by balazsorban44** [comment]
This issue has been automatically locked due to no recent activity. If you are running into a similar issue, please create a new issue with the steps to reproduce. Thank you.
